src/R/Components/Impl/PackageManager/Implementation/RPackageManager.cs

```diff
@@ -47,6 +47,87 @@ internal class RPackageManager : IRPackageManager {
             await RPackageWebParser.RetrievePackageInfo(uri, pkg);
         }
 
+        public void InstallPackage(string name, string libraryPath) {
+            string script;
+            if (string.IsNullOrEmpty(libraryPath)) {
+                script = string.Format("install.packages(\"{0}\")", name);
+            } else {
+                script = string.Format("install.packages(\"{0}\", lib=\"{1}\")", name, libraryPath.ToRPath());
```

Comment:
All the string literals inserted here and below should use `ToRStringLiteral` to ensure proper quoting for any kinds of weird names.